src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/test

```diff
@@ -0,0 +1,17 @@
+##
+## This test starts a sandbox, deploys a LagomJava service and executes an HTTP request
+## through the Service Gateway to ensure the service is reachable.
+##
+
+> sandbox run 2.0.2
+
+> install
+
+> verifyConductInfo
+
+> verifyIsStarted lagom-service-impl
+
+$ sleep 5000
+> assertRequest 9000 /foo foo-response-hardcoded in FooServiceImpl
```

Comment:
The `assertRequest` awaits until the response is the expected. In this case, the request is a `GET http://192.168.10.1:9000/foo` and the expected response is `foo-response-hardcoded in FooServiceImpl`.

Note that `GET` is the only method supported at the moment and that `192.168.10.1` is hardcoded in `build.sbt`.